server/routes/v1/user.ts

```diff
@@ -1,60 +1,62 @@
 import express, { Request, Response } from "express";
-import moment from "moment";
 import jwt from "jsonwebtoken";
 
 import EmailService from "../../services/emailService";
 import db from "../../models";
 import UserController from "../../controllers/v1/user";
-import { User } from "../../models/user";
 
 const userRouter = express.Router();
+const apiRoute = `${process.env.PAGE_URL}api/v1`;
+const cookie_key = "token";
 
 const uContr: UserController = new UserController(
   db.User,
-  new EmailService()
+  new EmailService(apiRoute)
 );
 
-let response;
-let user: User | undefined;
-
-async function signOutHandler(req: Request, res: Response) {
+async function signOut(req: Request, res: Response) {
   res
-    .clearCookie("token", {
+    .clearCookie(cookie_key, {
       httpOnly: true,
-      signed: true,
     })
-    .status(200)
-    .json({ message: "Signed out successfully" });
+    .status(204).json();
 }
 
 export async function signInHandler(req: Request, res: Response) {
   try {
     const { userName, password } = req.body;
 
     if (!userName || !password) {
-      res.status(400).json({ message: "Username or password is undefined" });
+      res.status(400).json({
+        message: !userName
+          ? !password
+            ? "Username and password"
+            : "Username"
+          : "Password" + " is undefined",
```

Comment:
```suggestion
          : "Password" + " not provided",
```

This makes it easier for users to understand.